airbyte-integrations/connectors/source-stripe/source_stripe/manifest.yaml

```diff
@@ -1487,7 +1487,7 @@ streams:
   # - State: Uses the event stream to call events related to streams (event types provided by params).
   - $ref: "#/definitions/streams/application_fees_refunds"
   - $ref: "#/definitions/streams/bank_accounts"
-  - $ref: "#/definitions/streams/invoice_line_items"
+#  - $ref: "#/definitions/streams/invoice_line_items" - Rollback to Python version due to
```

Comment:
nit: can we add the reference here?

Reply:
Yes, it was already included in the next commit.🙂